account_banking_ach_base/views/account_move.xml

```diff
@@ -5,7 +5,7 @@
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_account_invoice_filter" />
         <field name="arch" type="xml">
-            <filter name="unpaid" position="after">
+            <filter name="open" position="after">
```

Comment:
@nileshsheliya 
I would move any non-linter to 
git commit -m "[MIG] $MODULE: Migration to 15.0"